zone/bot_commands/precombat.cpp

```diff
@@ -18,7 +18,7 @@ void bot_command_precombat(Client* c, const Seperator* sep)
 		return;
 	}
 
-	if (!c->DoLosChecks(c->GetTarget())) {
+	if (RuleB(Bots, BotsRequireLoS) && !c->DoLosChecks(c->GetTarget())) {
```

Comment:
In a lot of the places you're checking for DoLosChecks you're also adding this RuleB. We could simplify this by doing 

```cpp
bool Mob::DoLosChecks(Mob* other) {
	if (IsBot() && !RuleB(Bots, BotsRequireLoS)) {
		return false;
	}
```